src/app/features/unit/card/unit-card.component.scss

```diff
@@ -0,0 +1,119 @@
+@use './src/styles/colors' as st;
```

Comment:
czy to nie byłoby  może czytelniejsze gdyby było `as colors` zamiast `as st`?

Reply:
zmieniłem